tests/remote_point_evaluation/setup_rpe_with_intersections_02.cc

```diff
@@ -100,15 +100,11 @@ do_test(const unsigned int n_quad_points)
       std::vector<bool>(),
       1.0e-9);
 
-  std::cout << "FINISHED" << std::endl;
-
   auto rpe_intersection_data =
     intersection_location
       .convert_to_distributed_compute_point_locations_internal(n_quad_points,
                                                                tria,
                                                                mapping);
-  std::cout << "WTF" << std::endl;
```

Comment:
:speak_no_evil:

Reply:
It seems I was pretty desperate at some point...